tests/test_xmlbuilder.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from apidaze.xmlbuilder import XMLBuilder
+from apidaze.xmlbuilder.nodes.answer import Answer
```

Comment:
I think we may get rid of the `nodes` folder and move all those node files to their parent folder to achieve an easier import statement for the SDK consumers;

`from apidaze.xmlbuilder import XMLBuilder, Answer, Speak`

If moving them are not desired, we can expose them by importing them in the file we would like to expose them from. For example, if the following snippet is added in `apidaze/xmlbuilder/__init__.py`, then one may import the builder and some nodes as in the snippet after the next;

```python
from apidaze.xmlbuilder.nodes.answer import Answer # NOQA
from apidaze.xmlbuilder.nodes.base_node import BaseNode
from apidaze.xmlbuilder.nodes.bind import Bind # NOQA
from apidaze.xmlbuilder.nodes.conference import Conference # NOQA
from apidaze.xmlbuilder.nodes.dial import Dial # NOQA
from apidaze.xmlbuilder.nodes.echo import Echo # NOQA
from apidaze.xmlbuilder.nodes.hangup import Hangup # NOQA
from apidaze.xmlbuilder.nodes.intercept import Intercept # NOQA
from apidaze.xmlbuilder.nodes.playback import Playback # NOQA
from apidaze.xmlbuilder.nodes.record import Record # NOQA
from apidaze.xmlbuilder.nodes.ringback import Ringback # NOQA
from apidaze.xmlbuilder.nodes.speak import Speak # NOQA
from apidaze.xmlbuilder.nodes.wait import Wait # NOQA
```
**Usage**
```python
import apidaze.xmlbuilder import XMLBuilder, Answer, Hangup
```

I wonder what you think about this?

Reply:
I like the idea, but it will generate linter errors for unused imports

Reply:
That's why I put `# NOQA` at the end of each line where we have unused imports. Would that help out?

Reply:
Yes :)